_includes/head.html

```diff
@@ -4,17 +4,17 @@
   <meta name="description" content="{% if page.excerpt %}{{ page.excerpt | strip_html | strip_newlines | truncate: 160 }}{% else %}{{ site.description }}{% endif %}">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <meta name="theme-color" content="#456BD9">
-  <meta name="msapplication-config" content="{{ site.url }}/browserconfig.xml">
-  <link rel="icon" type="image/png" href="{{ site.url }}/favicon-32x32.png" sizes="32x32">
-  <link rel="icon" type="image/png" href="{{ site.url }}/favicon-16x16.png" sizes="16x16">
-  <link rel="mask-icon" href="{{ site.url }}/safari-pinned-tab.svg" color="#456BD9">
+  <meta name="msapplication-config" content="browserconfig.xml">
+  <link rel="icon" type="image/png" href="favicon-32x32.png" sizes="32x32">
+  <link rel="icon" type="image/png" href="favicon-16x16.png" sizes="16x16">
+  <link rel="mask-icon" href="safari-pinned-tab.svg" color="#456BD9">
```

Comment:
Was it intentional to remove `site.url` occurrences?

Reply:
Yes. The netlify deploy previews would have all their links pointed to the production site otherwise, instead of in pages within the deploy preview. Also it would be pointing to the production images instead of the ones on the deploy preview.